stdlib/sys/fs.gr

```diff
@@ -0,0 +1,1068 @@
+module Fs
+
+include "sys/file"
+include "bytes"
+include "array"
+include "option"
+include "result"
+include "path"
+include "int64"
+include "string"
+include "runtime/wasi"
+include "runtime/unsafe/wasmi32"
+include "runtime/dataStructures"
+
+/**
+ * Potential errors that can be raised from system interactions.
+ */
+provide enum FileError {
```

Comment:
Do we want to pre-emptively add documentation for these, so its done when #992 is merged?

Reply:
I think most of the values from the types defined are self-explanatory so not sure if needed; I'll add doc comments where things may not be entirely clear though.